doc/setup_instructions.md

```diff
@@ -235,19 +246,19 @@ code editor.
 Thrive uses some external C# packages which need to be restored before
 compiling.
 
-On Linux open a terminal to the thrive folder and run the following
+On Linux, or if you're using visual studio code, open a terminal to the thrive folder and run the following
 command: `nuget restore` it should download the missing nuget
 packages. You may need to rerun this command when new package
 dependencies are added to Thrive. Note: if you use MonoDevelop it
 *might* automatically restore missing packages using nuget when
-compiling the game within MonoDevelop.
+compiling the game within MonoDevelop. 
 
 
-On Windows you should use Visual studio to restore the packages. To do
+On Windows you should use visual Studio to restore the packages. To do
```

Comment:
Why did you change this to be lower case? When referring to the software name uppercase should be used.

Reply:
Noticed you referred to Visual Studio in lower case mostly, wanted to keep it consistent, will change that too.